src/js/components/help/components/pages/communities/scenes/announcements/Announcements.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import styled from 'styled-components'
+import Sidebar from '../../../../help-topics'
+import Topic from '../../../../topic'
+import Desrciption from '../../../../help-description'
+import Description from '../../../../help-description';
+import {Link} from 'react-router-dom'
+
+
+const StyledAnnouncements = styled.div`
+.help-description-container{
+  display:flex;
+  margin 5px;
+  padding: 5px;
+}
+
+img{
+width:700px;
+height: 500px;
```

Comment:
Indenting here should be fixed.

Reply:
Also I noticed most of the help pages have similar styles. If that's the case, you should abstract the styling to a different component and just import that one component. We could discuss more on this later.